addons/web/static/src/js/chrome/abstract_action.js

```diff
@@ -27,6 +27,12 @@ var AbstractAction = Widget.extend(ActionMixin, {
      */
     hasControlPanel: false,
 
+    /**
+     * Owl component to render as the control panel. The component
+     * should extend the regular control panel component.
+     */
+    controlPanelComponent: ControlPanel,
```

Comment:
this commit needs a test and can be merged in master without waiting for spreadsheet

Reply:
test should create an action with a control panel that is not the default control panel